src/ucp/rndv/rndv.c

```diff
@@ -529,6 +531,7 @@ ucp_rndv_progress_rma_zcopy_common(ucp_request_t *req, ucp_lane_index_t lane,
     ucp_ep_h ep             = req->send.ep;
     uct_ep_h uct_ep         = ucp_ep_get_lane(ep, lane);
     ucp_ep_config_t *config = ucp_ep_config(ep);
+    size_t lanes_count      = ucs_popcount(req->send.rndv.zcopy.lanes_map_all);
```

Comment:
popcount is not converted to CPU instruction when not passing --enable-optimizations flag
so we should avoid it in fast path
maybe we can keep it in the request, it's not so big anyway

Reply:
```c
union {
    struct {
        /* Actual lanes map */
        ucp_lane_map_t lanes_map_all;

        /* Actual lanes count */
        uint8_t        lanes_count;
    } zcopy;

    struct {
        /* Data start offset of this request */
        size_t offset;
    } rtr;
};
```
That is the part of `ucp_request` where `lanes_count` is stored. If size of `ucp_lane_map_t` is less than 8, everything is OK since `lanes_count` + `lanes_map_all` fields took no more than `offset`, so union has 8 bytes size.

But if `ucp_lane_map_t` has 8 bytes, `lanes_count` became the 9th byte, so the union size became 16 bytes .

The only possible option that I see is to move that 1-byte field to another place inside `ucp_request` sturcture, but all other fields inside `struct rndv {...}` have 8-byte size, so on any place it would create 7 byte padding.

Is that so significant to have that value prestored in that field instead of `popcount` on that place? I mean that RMA flow which is often used for big messages and that's only for protov1, WDYT?

Reply:
ok, though maybe worth converting `ucs_assert_always(lanes_count > 0);` to `ucs_assert(ucs_popcount(lanes_count) > 0);` and remove the local var lanes_count, so it will be calculated only when needed in release mode

Reply:
Done.